components/Auth/AuthContent.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/prop-types */
```

Comment:
haha, how is prop types complaining? do you have the package installed? i noticed you're passing props, but this is a js file, so should be chill